src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCount = 40;
+  const totalSum = dailyRentalCount * days;
+  const discountFromTreeDays = 20;
+  const discountFromSevenDays = 50;
+
+  if (days >= 3 && days <= 6) {
```

Comment:
The condition `days <= 6` is redundant because if `days >= 7`, the previous `if` condition would have returned already, so this check is not necessary. This is in line with checklist items 4 and 5, which suggest simplifying conditions and avoiding unnecessary `else` after `if` with `return`.